### ISSUE TITLE
Tensorflow custom object training increases inference time compare to base model with transfer learning

### ISSUE THREAD

**Issue by asguradian** [tensorflow/models#5611]
I trained a custom object using tensorflow standard training proess with transfer learning.  After 1000 step i extracted the frozen inferene graph and test the model. Accuracy is good but the inference time is nearly double than the base model from which i did transfer learning. Is this the normal behaviour or something is wrong.  I followed the exact standard process to train  on my object. My base model is faster_rcnn_resnet101_coco which take 1.84 sec for inference while the the model I extracted after training took 3.54s. I use the exact same image to test, all configuration is same when i calculated inference time for base model and the mode l extracted

**Comment by asguradian** [comment]
I used tensorflow 1.12(latest), protoc(3), python3 faster_rcnn_inception_v2 and have not changed anything on config except  batch size to 1 and number of class to 1.

**Comment by dori-reichmann** [comment]
Inference time depends on input image size and batch size. Make sure you compare with both of these fixed.

**Comment by wt-huang** [comment]
In the original TensorFlow model, there is a device assigning function to optimize graph nodes on GPU/CPU which increases inference speed. This function is disabled when using the frozen inference graph, resulting the increased inference time.

**Comment by joei1994** [comment]
@wt-huang Could you explain how to enable those devices when export custom model
